spec/lib/seeds/course_builder_spec.rb

```diff
@@ -46,7 +46,7 @@
 
         expect { course_builder }
           .to change { existing_course.reload.title }.from('Development 101').to('Foundations')
-                                                     .and change { existing_course.position }.from(2).to(1)
+          .and change { existing_course.position }.from(2).to(1)
```

Comment:
Consequence of the Layout/MultilineMethodCallIndentation: Rubocop rule change

Reply:
Agree this is much nicer